src/services/edgesFile.js

```diff
@@ -1,58 +1,61 @@
-import fastJsonStringify from 'fast-json-stringify';
-import fs from 'fs';
 import path from 'path';
-
 import web3 from './web3';
+import fs from 'fs';
+const copyTo = require('pg-copy-streams').to;
```

Comment:
can we write this as an import?

Reply:
I tried but we are only importing `to` and with import I couldn't get the same behaivour

Reply:
It worked with [subpath imports](https://nodejs.org/api/packages.html) :)

Reply:
thanks!!